test/functional/tests/stats/test_consistency_between_outputs.py

```diff
@@ -0,0 +1,277 @@
+#
+# Copyright(c) 2020 Intel Corporation
+# SPDX-License-Identifier: BSD-3-Clause-Clear
+#
+
+import random
+import re
+import pytest
+
+from api.cas.cache_config import CacheMode, CacheLineSize, CacheModeTrait
+from api.cas.casadm import OutputFormat, print_statistics, start_cache
+from core.test_run import TestRun
+from storage_devices.disk import DiskType, DiskTypeSet, DiskTypeLowerThan
+from test_tools.dd import Dd
+from test_tools.disk_utils import Filesystem
+from test_utils.size import Size, Unit
+
+iterations = 64
+cache_size = Size(8, Unit.GibiByte)
+
+
+@pytest.mark.parametrize("cache_line_size", CacheLineSize)
+@pytest.mark.parametrize("cache_mode", CacheMode.with_any_trait(CacheModeTrait.InsertRead
+                                                                | CacheModeTrait.InsertWrite))
+@pytest.mark.parametrize("test_object", ["cache", "core"])
+@pytest.mark.require_disk("cache", DiskTypeSet([DiskType.optane, DiskType.nand]))
+@pytest.mark.require_disk("core", DiskTypeLowerThan("cache"))
+def test_output_consistency(cache_line_size, cache_mode, test_object):
+    """
+        title: Test consistency between different cache statistics' outputs.
+        description: |
+          Check if OpenCAS's statistics for cache are consistent regardless of the output format.
+        pass_criteria:
+          - Cache statistics in CSV format matches cache statistics in table format.
```

Comment:
update description - tests are now merged

Reply:
Thx, I forgot to update it.